Decisions/scs-XXXX-v1-requirements-for-container-registry.md

```diff
@@ -0,0 +1,364 @@
+|-------------------------------------|-----------------------------------------------|---------------------------------------------------------------------|------------------------------|
+| Audit Logs                          | ✓                                             | ✓                                                                   | ✗                            |
+| Authentication of system identities | ✓ Robot Accounts                              | ✓ Robot Accounts                                                    | ✗                            |
+| Authentication of users             | ✓ Local database, LDAP, OIDC, UAA             | ✓ Local database, LDAP, Keystone, JWT                               | ✓ Local database             |
+| Authorization                       | ✓                                             | ✓                                                                   | ✓                            |
+| Automation                          | ✓ Webhooks                                    | ✓ Webhooks, building images                                         | ✗                            |
+| Vulnerability scanning              | ✓ Trivy, Clair                                | ✓ Clair                                                             | ✗                            |
+| Content Trust and Validation        | ✓ Cosign                                      | ✓ Cosign                                                            | ✗                            |
+| Multi-tenancy                       | ✓ (not on the storage level)                  | ✓ (not on the storage level)                                        | ✓ (not on the storage level) |
+| Backup and restore                  | ✓                                             | ✓                                                                   | ✗                            |
+| Monitoring                          | ✓ Prometheus metrics                          | ✓ Prometheus metrics                                                | ✓ Prometheus metrics         |
+| HA mode                             | ✓                                             | ✓                                                                   | ✗                            |
+| Registry replication                | ✓                                             | ✓                                                                   | ✓                            |
+| Proxy cache                         | ✓                                             | ✓ Feature is in the technology preview stage (non production ready) | ✗                            |
+| Quota management                    | ✓ Based on storage consumption or image count | ✓ Based on storage consumption                                      | ✗                            |
```

Comment:
Harbor is storage consumption  only

Reply:
Thank you for pointing this out @Vad1mo. I found that it is mentioned in the v1.10 [docs](https://goharbor.io/docs/1.10/administration/configure-project-quotas/) but it was removed in the v2.0.0 [release](https://github.com/goharbor/harbor/releases/tag/v2.0.0).

Reply:
I incorporated it in 02382f14b5ba7204845eb3e690c4d5b78a49770b